extension/src/experiments/webview/messages.ts

```diff
@@ -156,7 +156,7 @@ export class WebviewMessages {
 
   private setExperimentStars(ids: string[]) {
     this.experiments.toggleStars(ids)
-    this.notifyChanged()
+    this.sendWebviewMessage()
```

Comment:
[F] Stars are not shown in any of the trees so we only need to update the webview.